peekingduck/commands/core.py

```diff
@@ -0,0 +1,204 @@
+    else:
+        start_time = perf_counter()
+        runner = Runner(
+            pipeline_path=pipeline_config_path,
+            config_updates_cli=node_config,
+            custom_nodes_parent_subdir=nodes_parent_dir,
+            num_iter=num_iter,
+        )
+        end_time = perf_counter()
+        logger.debug(f"Startup time = {end_time - start_time:.2f} sec")
+        runner.run()
+
+
+@click.command()
+def verify_install() -> None:
+    """Verifies PeekingDuck installation by running object detection on
+    'wave.mp4'.
+    """
+    LoggerSetup.set_log_level("info")
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        pipeline_config_path = Path(tmp_dir) / "verification_pipeline.yml"
+
+        _create_pipeline_config_yml(
+            [
+                {
+                    "input.visual": {
+                        "source": "https://storage.googleapis.com/peekingduck/videos/wave.mp4"
+                    }
+                },
+                "model.yolo",
+                "draw.bbox",
+                "output.screen",
+            ],
+            pipeline_config_path,
+        )
+
+        runner = Runner(
+            pipeline_path=pipeline_config_path,
+            config_updates_cli="None",
+            custom_nodes_parent_subdir="src",
+            num_iter=None,
+        )
+        runner.run()
+
+
+def _create_custom_folder(custom_folder_name: str) -> None:
+    """Makes custom nodes folder to create custom nodes.
+
+    Args:
+        custom_folder_name (:obj:`str`): Name of the custom nodes folder.
+    """
+    curr_dir = _get_cwd()
+    custom_nodes_dir = curr_dir / "src" / custom_folder_name
+    custom_nodes_config_dir = custom_nodes_dir / "configs"
+
+    logger.info(f"Creating custom nodes folder in {custom_nodes_dir}")
+    custom_nodes_dir.mkdir(parents=True, exist_ok=True)
+    custom_nodes_config_dir.mkdir(parents=True, exist_ok=True)
+
+
+def _create_pipeline_config_yml(
+    default_nodes: List[Union[str, Dict[str, Any]]] = None,
+    default_path: Path = Path("pipeline_config.yml"),
+) -> None:
+    """Initializes the declarative *pipeline_config.yml*.
+
+    Args:
+        default_nodes (List[Union[str, Dict[str, Any]]]): A list of PeekingDuck
+            node. For nodes with custom configuration, use a dictionary instead
+            of a string.
+        default_path (Path): Path of the pipeline config file.
+    """
+    # Default yml to be discussed
+    if default_nodes is None:
+        default_nodes = [
+            {
+                "input.visual": {
+                    "source": "https://storage.googleapis.com/peekingduck/videos/wave.mp4"
+                }
+            },
+            "model.posenet",
+            "draw.poses",
+            "output.screen",
+        ]
+    default_yml = dict(nodes=default_nodes)
+
+    with open(default_path, "w") as yml_file:
+        yaml.dump(default_yml, yml_file, default_flow_style=False)
+
+
+def _get_cwd() -> Path:
+    return Path.cwd()
```

Comment:
Is there a need to re-define `Path.cwd()` as `_get_cwd()` instead of using it directly?

Reply:
remanent of the old code. I assume this was to provide a consistent cwd for the various functions. But since it's just calling `Path.cwd()`, I've removed it until the need to have such a function arise again.